src/Panel/Panel/Panel.less

```diff
@@ -1,12 +1,10 @@
-@import '../../style/variables.less';
-
-@handlestyle: 5px solid fade(@primary-color, 50%);
+@handlestyle: 5px solid var(--ant-primary-4);
```

Comment:
I'd suggest to transform this into a css custom property too.

Reply:
Actually the value of `--ant-primary-4` is automatically generated based on the given primary color. Thus the value is more or less guaranteed to match the desired color scheme.